src/target/riscv/riscv.c

```diff
@@ -611,13 +622,43 @@ static int jim_report_ebreak_config(const struct riscv_private_config *config,
 	return JIM_OK;
 }
 
+static int jim_configure_cetrig(struct riscv_private_config *config,
+		struct jim_getopt_info *goi)
+{
+	if (goi->argc == 0) {
+		Jim_WrongNumArgs(goi->interp, 1, goi->argv - 1,
+				"?disable?or?enable?");
+		return JIM_ERR;
+	}
+
+	/* Here a dcsr field is processed, e.g: riscv.cpu configure -mprven enable" */
+	struct jim_nvp *opt_nvp;
+	if (jim_getopt_nvp(goi, nvp_cetrig_opts, &opt_nvp) != JIM_OK) {
+		jim_getopt_nvp_unknown(goi, nvp_cetrig_opts, /*hadprefix*/ true);
+		return JIM_ERR;
+	}
+	config->dcsr_cetrig = opt_nvp->value;
+	return JIM_OK;
+}
+
+static int jim_report_cetrig_config(const struct riscv_private_config *config,
+		Jim_Interp *interp)
+{
+	const char * const cetrig_opt = jim_nvp_value2name_simple(nvp_cetrig_opts,
```

Comment:
```suggestion
	const char *cetrig_opt = jim_nvp_value2name_simple(nvp_cetrig_opts,
```
I'd suggest dropping this `const`. AFAIU, the styleguide will be updated sometime to include this rule.
See https://review.openocd.org/c/openocd/+/8750/comment/984b8414_f6f804e4/

Reply:
Addressed